README.md

```diff
@@ -43,24 +54,179 @@ The plugin will remove the custom attribute from the element and replace its con
 You can use external libraries for this as well, no problem. Just make sure you are passing in a function that takes a string and returns a string. You might have to wrap the library function if it doesn't behave like this, but it will work with anything that transforms content.
 
 ```html
-<p md>Wow, it's **markdown**!</p>
+<p name>Transform me!</p>
 ```
 
 ```js
-const MarkdownIt = require('markdown-it')
-const md = new MarkdownIt() // can pass options here
-const content = require('posthtml-content')({
-  md: md.renderInline.bind(md)
+const fn = require('fn')
+
+const plugin = require('posthtml-content')({
+  name: (ctx) => fn(ctx)
+})
+
+posthtml([plugin]).process(html)
+```
+
+```js
+const fn = require('fn')
+
+const plugin = require('posthtml-content')({
+  name: fn.bind(fn)
+})
+
+posthtml([plugin]).process(html)
+```
+
+```html
+<p>👍</p>
+```
+
+## Examples
+
+#### Markdown
+
+```html
+<p md>Wow, it's **Markdown**!</p>
+```
+
+```js
+const markdown = require('markdown-it')(/* options */)
+
+const plugin = require('posthtml-content')({
+  md: (md) => markdown.renderInline(md)
+})
+
+posthtml([plugin]).process(html)
+```
+
+```html
+<p>Wow, it's <strong>Markdown</strong>!</p>
+```
+
+#### PostCSS
+
+```sugarss
+<style postcss>
+  .test
+    text-transform: uppercase;
+
+    &__hello
+      color: red;
+
+    &__world
+      color: blue;
+</style>
+```
+
+```js
+const postcss = require('postcss')([ require('postcss-nested')() ])
+const options = { parser: require('sugarss'), map: false }
+
+const plugin = require('posthtml-content')({
+  postcss: (css) => postcss.process(css, options).css
+})
+
+posthtml([plugin]).process(html)
+
+```
+
+```html
+<style>
+  .test {
+    text-transform: uppercase;
+  }
+
+  .test__hello {
+    color: red;
+  }
+
+  .test__world {
+    color: blue;
+  }
+</style>
+```
+
+#### Babel
+
+```html
+<script babel>
+  const hello = 'Hello World!'
+  let greeter = {
+    greet (msg) { alert (msg) }
+  }
+  greeter.greet(hello)
+</script>
+```
+
+```js
+const babel = require('babel-core')
+const options = { presets: ["es2015"], sourceMaps: false }
+
+const plugin = require('posthtml-content')({
+  babel: (js) => babel.transform(js, options).code
 })
 
 posthtml([plugin]).process(html)
 ```
 
 ```html
-<p>Wow, it's <strong>markdown</strong>!</p>
+<script>
+  'use strict';
+  var hello = "Hello World!";
+  var greeter = {
+    greet: function greet (msg) {
+      alert(msg);
+    };
+  };
+  greeter.greet(hello);
+</script>
```

Comment:
i'd like for the examples to follow https://github.com/feross/standard as our codebase does